module3/lessons/Week4/LinkedIn.md

```diff
@@ -0,0 +1,158 @@
+* **Action Item:** "Reach me at email@address.com or book time on my calendar here: [Calendar link]."
+
+### Section 5: Experiences and Recommendations
+
+#### Experiences
+This section is where you showcase your current and prior work:
+
+* What your role was/is
+* What impact you had
+* Mention your specialties and how this prepared you for future roles
+
+Title - Action - Benefit
+
+Short description of the company
+
+Details about your role
+
+Accomplishments, results, highlights
+* Include skills relevant to your future role as a developer
+* Link the company when adding roles
+
+
+#### Endorsements & Recommendations
+* Endorsements allow other people in your network to confirm and highlight your skills
+* Recommendations are a way for you to showcase praise you’ve received or given to others
+
+#### Endorsement Request Example
+
+"If you were happy with the work I performed at Widgets, Inc, would you be so kind as to endorse me on LinkedIn?
+
+The skills and expertise I would like to be endorsed for are:
+Project Management, Leadership Team Building, and SEO.
+
+All you need to do is visit my profile and scroll down to the Skills and Expertise section located under my Summary.
+
+Here's the link to my profile: [link here]
+
+Thanks!"
+
+### Section 6: Making Connections
+
+A key part of LinkedIn is building your network through making connections. The key thing to note here is **include a note when you send a connection request**. Especially if it's someone you don't know well, it will make it much more likely that that person connects with you.
+
+### Additional Resources
+* How to Take Your Own Linkedin Photo
+   * [Blog Post from Turing](https://writing.turing.edu/how-to-take-your-own-linkedin-headshot/)
+   * [Article by Michele LaCagnina](https://www.linkedin.com/pulse/how-take-painless-free-linkedin-headshot-michele-lacagnina/)
+* [Unsplash.com](https://unsplash.com) for high-resolution images for your profile’s cover photo
+* [How to Write a LinkedIn Summary](https://www.jobscan.co/blog/linkedin-summary-examples/) Article by Jon Shields
+* [13 Creative LinkedIn Summary Examples & How to Write Your Own](https://blog.hubspot.com/sales/linkedin-summary-examples) Article by Aja Frost
+* [How to Analyze Your Linkedin Profile Using the Linkedin Dashboard](https://www.socialmediaexaminer.com/how-to-analyze-linkedin-profile-using-linkedin-dashboard/) Article by Luan Wise
+
+
+## Deliverable
+
+**The first draft of your Linkedin Profile is Due this Thursday at 4pm.**
+
+To turn it in, send a link to your LinkedIn profile in the chat with your instructors.
+
+At a minimum, your profile should include:
+* Professional-looking headshot
+* Customized Cover Photo
+* Updated headline statement
+* Summary statement using your professional story
+* Turing added to Education section
+    * You should also add Turing to the Experience section *if* you do not have previous professional experience
+* At least 10 connections

```

Comment:
Do you want to add our LinkedIn profiles to this list? That gives them 3-4 to start with. Maybe even have them share with each other (or in a Slack thread), that's 9 more. Maybe bump this to 20 since we're spotting them more than 10 LOL

Reply:
Good call! I'll put an instructor note to do a sharing LinkedIn threads and have the instructor post instructor LinkedIns too. 

With that I'm very down to bump it to 20, I think it's reasonable to require them to reach out to some folks outside of class.